config/notifies.go

```diff
@@ -15,62 +15,50 @@ package config
 
 import (
 	"fmt"
+	"strings"
 )
 
 var (
 	// DefaultHipchatConfig defines default values for Hipchat configurations.
 	DefaultHipchatConfig = HipchatConfig{
-		ColorFiring:   "purple",
-		ColorResolved: "green",
+		Color:         `{{ if eq .Status "firing" }}purple{{ else }}green{{ end }}`,
 		MessageFormat: HipchatFormatHTML,
 	}
 
 	// DefaultSlackConfig defines default values for Slack configurations.
 	DefaultSlackConfig = SlackConfig{
-		ColorFiring:   "warning",
-		ColorResolved: "good",
-
-		Templates: SlackTemplates{
-			Title:     "slack.default.title",
-			TitleLink: "slack.default.title_link",
-			Pretext:   "slack.default.pretext",
-			Text:      "slack.default.text",
-			Fallback:  "slack.default.fallback",
-		},
+		Color:     `{{ if eq .Status "firing" }}warning{{ else }}good{{ end }}`,
+		Title:     `{{template "slack.default.title" . }}`,
```

Comment:
I see where you are going.
I actually like this approach – it's a clean way to mix between inlining and files.